### ISSUE TITLE
Network interface script

### ISSUE THREAD

**Issue by geasus** [la-cc/hetzner-proxmox-single#8]
I am following your guide exactly and when i run the bash script for the network interfaces configuration it ask for a name for the network interface which i can not find in the guide but everytime when i try to run the bash script i lose all my access to the server, proxmoxx wont be reachable at 8006 anymore and SSH also not working. No idea what i am missing or doing different then the guide.

**Comment by sollevix67** [comment]
The same here

**Comment by HansenBerlin** [comment]
Yep, works fine until running the script. Then the machine is bricked.

**Comment by naghidragos** [comment]
choose the enp0s..etc interface instead of eth0, the one that shows up in ifconfig at the begining. you can also check in pve proxmox screen which one is active one. the one that has the main ip address from hetzner assigned.

**Comment by la-cc** [comment]
Check first the interface with ifconfig, it can be different from the example. 
If you using the wrong interface and configure it, you will break things! 

You can see here an example from my blog under point [3. Setup network config](https://medium.com/@artem_lajko/setup-ve-with-proxmox-on-hetzner-single-mode-6b76061efcdb)
